front/src/applications/stdcm/components/StdcmResults/StdcmResultsTable.tsx

```diff
@@ -6,19 +6,22 @@ import { useTranslation } from 'react-i18next';
 
 import type {
   StdcmResultsOperationalPointsList,
+  StdcmSimulation,
   StdcmSuccessResponse,
 } from 'applications/stdcm/types';
 import { getStopDurationTime } from 'applications/stdcm/utils/formatSimulationReportSheet';
 
 type SimulationTableProps = {
   stdcmData: StdcmSuccessResponse;
+  stdcmSimulation: StdcmSimulation;
```

Comment:
As we use only the mass here, we don't need to pass all this object in props

```suggestion
  selectedSimulationTotalMass?: number;
```

Reply:
in the future we will pass more parameters, but i can pass only the input field